README.md

```diff
@@ -353,6 +353,18 @@ python setup.py develop
 
 ```
 
+**Intel GPU builds**
+
+In this mode PyTorch with Intel GPU support will be built.
+
+Please make sure [the common prerequisites](#prerequisites) as well as [the prerequisites for Intel GPU](#intel-gpu-support) are properly installed and the environmental variables are configured in prior. For build tool support, `Visual Studio 2022` is required.
```

Comment:
environmental -> environment 

Also, not sure what do you mean by `For build tool support`, but leaving it as is
```suggestion
Please make sure [the common prerequisites](#prerequisites) as well as [the prerequisites for Intel GPU](#intel-gpu-support) are properly installed and the environment variables are configured prior to starting the build. For build tool support, `Visual Studio 2022` is required.
```